### PR TITLE
Remove Bootstrap3

### LINKED ISSUE #1065: update node.js install in docker build
## What issue are you seeing?
When building the docker image the following message appears:

```
================================================================================
#7 3.257 ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
#7 3.257 ================================================================================
#7 3.257 
#7 3.257                            SCRIPT DEPRECATION WARNING                    
#7 3.257 
#7 3.257   
#7 3.257   This script, located at https://deb.nodesource.com/setup_X, used to
#7 3.257   install Node.js is deprecated now and will eventually be made inactive.
#7 3.257 
#7 3.257   Please visit the NodeSource distributions Github and follow the
#7 3.257   instructions to migrate your repo.
#7 3.257   https://github.com/nodesource/distributions
#7 3.257 
#7 3.257   The NodeSource Node.js Linux distributions GitHub repository contains
#7 3.257   information about which versions of Node.js and which Linux distributions
#7 3.257   are supported and how to install it.
#7 3.257   https://github.com/nodesource/distributions
#7 3.257 
#7 3.257 
#7 3.257                           SCRIPT DEPRECATION WARNING
#7 3.257 
#7 3.257 ================================================================================
```
our dockerfile loads a script from https://deb.nodesource.com/setup_16.x to install node.js
The https://github.com/nodesource/distributions site seems to have the new way to install node.js

### PR DESCRIPTION
**Pull request by b-meson**
## Fixes issue

Migrates off bootstrap3 and into bootstrap5. Thank you @psjalltheway !!! 

## Description of Changes

Removes deprecated files which are causing conflicts for the new HTML/CSS which will come in a separate PR. 

## Tests and Linting
- [x] This branch is up-to-date with the `develop` branch.
- [x] `pytest` passes on my local development environment.
- [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

**Review by sea-kelp — APPROVED** [review]
I had to remove the node.js/sass build logic since the index.scss file is no longer used. Looks good to me!

File: Makefile [hunk 1]
```diff
@@ -31,10 +31,6 @@ create_db: start
 	## Creating database
 	docker compose run --rm web alembic --config=./OpenOversight/migrations/alembic.ini stamp head
 
-.PHONY: assets
```
Comment (michplunkett) [inline Makefile:34]: 🥳 🥳 🥳 🥳

File: dockerfiles/web/Dockerfile-dev [hunk 2]
```diff
@@ -12,28 +12,9 @@ ENV PYTHONDONTWRITEBYTECODE=1
 RUN apt-get update && apt-get install -y xvfb libpq-dev python3-dev && \
     apt-get clean
 
-# install node
-ENV NODE_SETUP_SHA=5d07994f59e3edc2904c547e772b818d10abb066f6ff36ab3db5d686b0fe9a73
-RUN curl ${CURL_FLAGS} \
-      https://raw.githubusercontent.com/nodesource/distributions/b8510857fb4ce4b023161be8490b00119884974c/deb/setup_12.x
```
Comment (michplunkett) [inline dockerfiles/web/Dockerfile-dev:18]: I'm so happy to see this removed.

**Review by michplunkett — APPROVED** [review]
This is great! The only thing I'd want to make sure is that we have people available to roll it back should we need to when we push to production.

**Pull request merged**